CHANGELOG/CHANGELOG-1.27.md

```diff
@@ -593,20 +593,21 @@ The cause PR is [reverted](https://github.com/kubernetes/kubernetes/pull/117194)
   (The feature became GA in 1.23 and the gate was removed for all other
   components several releases ago.) If you were still manually
   enabling it you must stop now.' ([#116255](https://github.com/kubernetes/kubernetes/pull/116255), [@danwinship](https://github.com/danwinship))
- - Give terminal phase correctly to all pods that will not be restarted. 
+- Give terminal phase correctly to all pods that will not be restarted. 
```

Comment:
nit: does that need a tab? the previous bullet have another indentation

Reply:
Those two release notes don't seem to be related at all, one is about IPv6, another is about failed pods, so I don't think we need a tab here.